app/webpack/stylesheets/openstudios/_container_header.scss

```diff
@@ -0,0 +1,30 @@
+.container-header {
+  border-bottom: 2px solid $gto_gray;
+  width: 100%;
+  padding: 7px $openstudios-main-margin;
+}
+.container-header__brand {
+  float: left;
```

Comment:
this feels like a strange mix of float and flex and positioning.  Do we need all that?

Reply:
Yeah... I fought with flex box. I'm floating left and positioning fixed so that the centering of the `Open Studios Catalog` is calculated off the width of the page, not the center from the end of the logo to the right end of the page.